src/command/account_manager.rs

```diff
@@ -149,9 +149,7 @@ pub async fn restore_command(
         .with_secret_manager(secret_manager)
         // Will be overwritten by the backup's value.
         .with_client_options(
-            ClientOptions::new()
-                .with_node("http://localhost:14265")?
-                .with_node_sync_disabled(),
+            ClientOptions::new().with_node("http://localhost:14265")?, // .with_node_sync_disabled(),
```

Comment:
```suggestion
            ClientOptions::new().with_node("http://localhost:14265")?, // .with_ignore_node_health(),
```

Reply:
No harm to leave it there, might be good to enable it at some point to quickly test something